test/test-watcher-cross-stop.c

```diff
@@ -59,6 +59,10 @@ static void close_cb(uv_handle_t* handle) {
 
 
 TEST_IMPL(watcher_cross_stop) {
+#if defined(__MVS__)
+  RETURN_SKIP("IBM WORK ITEM: 130119");
```

Comment:
The RETURN_SKIP is fine, it's that an internal work item isn't useful.  If you replace it with a more general message, it should be fine.